include/extensions.h

```diff
@@ -11,6 +11,8 @@ struct background_config {
         wlc_resource surface;
         // we need the wl_resource of the surface in the destructor
         struct wl_resource *wl_surface_res;
+        // used to determine if client is a background
+	struct wl_client *client;
```

Comment:
This file does not follow the formatting rules, it uses spaces instead of tabs. The two lines I added do use 8 space wide tabs. I'm not sure why it looks like their indention is less than other lines here.  Should I use spaces in this file spaces in this file to be consistent.